ui.tests/test-module/specs/datepicker/datepicker.runtime.spec.js

```diff
@@ -184,6 +184,13 @@ describe("Form Runtime with Date Picker", () => {
             cy.get(`#${datePicker7}`).find("input").focus().should("have.value","2/8/2023");
 
         });
+
+        // check clear option
+        cy.get(`#${datePicker7}`).find(".datepicker-calendar-icon").should("be.visible").click().then(() => {
```

Comment:
Can we fix the datepicker-calendar-icon as per BEM,

`cmp-adaptiveform-datepicker__calendar-icon`